src/main/java/com/microsoft/graph/concurrency/ChunkedUploadResponseHandler.java

```diff
@@ -98,35 +101,36 @@ public void configConnection(@Nonnull final Response response) {
 		if (response.code() >= HttpResponseCode.HTTP_CLIENT_ERROR) {
 			logger.logDebug("Receiving error during upload, see detail on result error");
 
-			return new ChunkedUploadResult<UploadType>(
+			return new ChunkedUploadResult<>(
 					GraphServiceException.createFromResponse(request, null, serializer,
-						response, logger));
-		} else if (response.code() >= HttpResponseCode.HTTP_OK
+							response, logger));
+		} else if (response.code() >= HTTP_OK
 				&& response.code() < HttpResponseCode.HTTP_MULTIPLE_CHOICES) {
 			final Map<String, String> headers = responseHeadersHelper.getResponseHeadersAsMapStringString(response);
 			final String contentType = headers.get(Constants.CONTENT_TYPE_HEADER_NAME);
 			final String location = headers.get("Location");
-			if(contentType != null
-				&& contentType.contains(Constants.JSON_CONTENT_TYPE)) {
+			if (contentType != null
+					&& contentType.contains(Constants.JSON_CONTENT_TYPE)) {
 				try (final InputStream in = new BufferedInputStream(response.body().byteStream())) {
 					final String rawJson = CoreHttpProvider.streamToString(in);
 					final IUploadSession session = serializer.deserializeObject(rawJson, uploadSessionClass);
```

Comment:
shouldn't this be using the input stream instead?

Reply:
That's correct, I have either reverted too much code here or simply forgot to fix this place.